### PR TITLE
USHIFT-267: address comments from #50054

### PR DESCRIPTION
**Pull request by dhellmann**
Version(s):
  * MicroShift 4.12

Issue: [USHIFT-267](https://issues.redhat.com//browse/USHIFT-267)

QE review:

  These are not substantive changes to the content.

Additional information:

  We merged the outline in #50054 with some comments unresolved. This
  PR addresses the remaining issues.

  * fix symlinks in microshift_support
  * grammar and style fixes
  * itemize procedures
  * move procedures into modules
  * fix yaml syntax highlighting

/cc @jldohmann @kalexand-rh @ShaunaDiaz

### CODE REVIEW

**Comment by dhellmann** [comment]
I pushed an update to fix the title levels of the new modules.

File: modules/microshift-version-api.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+.Procedure
+
+* Using the API, access the `kube-public/microshift-version` ConfigMap to retrieve the same information.
+
+[source,terminal]
+----
+$ oc get configmap -n kube-public microshift-version -o yaml
+----
+
```
Comment (dhellmann) [inline modules/microshift-version-api.adoc:16]: @jldohmann in
https://github.com/openshift/openshift-docs/pull/50054#discussion_r1010985701 you say that we need a `+` between these parts of the procedure, and I can't quite work out how to add it properly.

If I make line 16 contain only `+` then it breaks the way the "Example output" is rendered.

If I add a line with `+` and a blank line before and after, I see the `+` in output.

Can you help me with the right syntax, please?

**Comment by ShaunaDiaz** [comment]
/lgtm 

Since I am waiting on these merges to complete, which I hope is today! I will review and update anything I need to once merged (with other PRs).

**Review by ShaunaDiaz — COMMENTED** [review]
/lgtm

File: modules/microshift-version-api.adoc (hunk 1 above)
Comment (kalexand-rh) [reply modules/microshift-version-api.adoc:16]: A `+` on both line 11 and line 16 should work.

Comment (ShaunaDiaz) [reply modules/microshift-version-api.adoc:16]: I can fix those in a follow-up PR.

**Comment by kalexand-rh** [comment]
@ShaunaDiaz, please reference this PR when you open the follow-up.

**Pull request merged**